dashboard/app/controllers/courses_controller.rb

```diff
@@ -163,7 +163,11 @@ def i18n_params
     # When the url of a course family is requested, redirect to a specific course version.
     if UnitGroup.family_names.include?(params[:course_name])
       unit_group = UnitGroup.latest_stable_version(params[:course_name])
-      redirect_to action: params[:action], course_name: unit_group.name if unit_group
+      if unit_group
+        redirect_path = url_for(action: params[:action], course_name: unit_group.name)
```

Comment:
According to [apidock](https://apidock.com/rails/ActionController/Redirecting/redirect_to), `redirect_to` will use `url_for` under the hood, so this shouldn't be a functional change.